utility/osp_deploy.sh

```diff
@@ -21,32 +23,50 @@ sudo yum install -y libguestfs-tools-c wget
 
 if [ -n "$1" -a -f "$1" ]; then
     mkdir newpkgs
```

Comment:
Will this fail, if the `newpkgs` already exists?

Reply:
Will add check before mkdir.

Reply:
"mkdir -p" should work in this case.